README.md

```diff
@@ -217,10 +217,14 @@ Next, create an Azure Service Principal, which is a special type of account that
 1. Create a Service Principal with a "contributor" role that is allowed to make changes to any resources in that subscription. Replace $AZURE_SUBSCRIPTION_ID with the ID you found in step 1 and run this command:
 
 ```
-az ad sp create-for-rbac --name "CICD" --role contributor --scopes /subscriptions/$AZURE_SUBSCRIPTION_ID --sdk-auth
+az ad sp create-for-rbac  --sdk-auth --name "github-deployer" --role contributor --scopes /subscriptions/$AZURE_SUBSCRIPTION_ID
```

Comment:
Also, I moved --sdk-auth to beginning of command to make it easier to paste the subscription ID at the end. The --sdk-auth parameter does result in a deprecation warning right now, but I'm assuming it's still necessary?

Reply:
This looks better with that flag moved into the beginning. The deprecation is unfortunate but required for this to work :(